lib/matplotlib/animation.py

```diff
@@ -921,28 +1318,42 @@ def to_html5_video(self):
             # Now open and base64 encode
             with open(f.name, 'rb') as video:
                 vid64 = base64.encodebytes(video.read())
-                self._base64_video = vid64.decode('ascii')
-                self._video_size = 'width="{0}" height="{1}"'.format(
-                        *writer.frame_size)
+                vid_len = len(vid64)
+                if vid_len >= embed_limit:
+                    import warnings
+                    warnings.warn('Animation movie is {0} bytes, exceeding '
+                            'the limit of {1}. If you\'re sure you want a '
+                            'large animation embedded, set the '
+                            'animation.embed_limit rc parameter to a larger '
+                            'value (in MB).'.format(vid_len, embed_limit))
+                else:
+                    self._base64_video = vid64.decode('ascii')
+                    self._video_size = 'width="{0}" height="{1}"'.format(
+                            *writer.frame_size)
 
             # Now we can remove
             os.remove(f.name)
 
-        # Default HTML5 options are to autoplay and to display video controls
-        options = ['controls', 'autoplay']
+        # If we exceeded the size, this attribute won't exist
+        if hasattr(self, '_base64_video'):
+            # Default HTML5 options are to autoplay and display video controls
+            options = ['controls', 'autoplay']
+
+            # If we're set to repeat, make it loop
+            if hasattr(self, 'repeat') and self.repeat:
+                options.append('loop')
 
-        # If we're set to repeat, make it loop
-        if self.repeat:
-            options.append('loop')
-        return VIDEO_TAG.format(video=self._base64_video,
-                                size=self._video_size,
-                                options=' '.join(options))
+            return VIDEO_TAG.format(video=self._base64_video,
+                                    size=self._video_size,
+                                    options=' '.join(options))
```

Comment:
No, it was indented so that we only return the tag if the base64-encoded video data was created (i.e. not too large). The mistake was not to have an alternative return if things go wrong. Will fix.